tests/misra/checkers.txt

```diff
@@ -193,440 +193,7 @@ Yes  CheckType::checkSignConversion
 Yes  CheckType::checkTooBigBitwiseShift
 Yes  CheckUninitVar::check
 Yes  CheckUninitVar::valueFlowUninit
-No   CheckUnusedFunctions::check                               require:unusedFunction
```

Comment:
The diff is a bit weird - the second variant used to start at line 440 at now is removed.  
Basically now, instead of checking F4 and H7 separately, they are checked together using cppcheck configuration combination capability (which is enforced with... `--force` argument ).